src/components/table/DataTable.svelte

```diff
@@ -15,8 +15,8 @@ setContext('scrollLeft', scrollLeft);
 setContext('scrollTop', scrollTop);
 
 function updateScroll() {
-  $scrollLeft = container.scrollLeft;
-  $scrollTop = container.scrollTop;
+  scrollLeft.set(container.scrollLeft);
+  scrollTop.set(container.scrollTop);
```

Comment:
Nice. I find this easier to read, assuming the top is valid at all, which I'm honestly forgetting right now.

Reply:
they're valid but if I were to  make an eslint set for svelte I would ban autosubscriptiions for setting. Just much easier to read!